DatadogSessionReplay/Tests/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/NodeRecorders/UIImageViewRecorderTests.swift

```diff
@@ -49,35 +47,8 @@ class UIImageViewRecorderTests: XCTestCase {
         // Then
         let semantics = try XCTUnwrap(recorder.semantics(of: imageView, with: viewAttributes, in: .mockAny()))
         XCTAssertTrue(semantics is SpecificElement)
-        DDAssertReflectionEqual(semantics.subtreeStrategy, .record, "Image view's subtree should be recorded")
-
-        let builder = try XCTUnwrap(semantics.wireframesBuilder as? UIImageViewWireframesBuilder)
-        XCTAssertEqual(builder.buildWireframes(with: WireframesBuilder()).count, 2)
-    }
-
-    func testWhenImageViewHasImageOrAppearance() throws {
```

Comment:
This test was duplicating configuration and assertions from the tests defined above. Getting rid of it 🧽.